src/sentry/static/sentry/app/components/u2finterface.jsx

```diff
@@ -94,10 +120,37 @@ const U2fInterface = React.createClass({
     );
   },
 
+  renderFailure() {
+    let {deviceFailure} = this.state;
+    return (
+      <div className="failure-message">
+        <p><strong>{t('Error: ')}</strong> {{
+          'DEVICE_ERROR': t('Your U2F device reported an error.'),
+          'DUPLICATE_DEVICE': t('This device is already in use.'),
+          'UNKNOWN_DEVICE': t('The device you used for sign-in is unknown.'),
+          'BAD_APPID': t('The Sentry server administrator modified the ' +
```

Comment:
This error message isn't very clear. Unless there are other ways of doing this, can we say something about the domain changing? "device registrations" isn't very clear to me what changed or how I'd fix the problem.